packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -46,6 +46,7 @@ function Wayfinding({ onStartDirections, onBack, location, onSetSize, isActive,
     /** Check if search results have been found */
     const [hasSearchResults, setHasSearchResults] = useState(true);
 
+    /** Check if the searched route throws errors */
```

Comment:
I don't think this useState is a "check", but more an "indicator".

The same applies to the one above.

Reply:
✅